apps/src/music/progress/MusicValidator.ts

```diff
@@ -30,16 +31,41 @@ export default class MusicValidator extends Validator {
   constructor(
     private readonly getIsPlaying: () => boolean,
     private readonly getPlaybackEvents: () => PlaybackEvent[],
+    private readonly getExemplarPlaybackEvents: () => PlaybackEvent[],
     private readonly getValidationTimeout: () => number,
     private readonly player: MusicPlayer,
     private readonly getPlayingTriggers: () => PlayingTrigger[],
+    private readonly getExemplarSettings: () => ExemplarSettings | undefined,
     private readonly conditionsChecker: ConditionsChecker = new ConditionsChecker(
       Object.values(MusicConditions).map(condition => condition.name)
     )
   ) {
     super();
   }
 
+  shouldValidateWithExemplar(): boolean {
+    const isEditingExemplar = getAppOptionsEditingExemplar();
+    return (
+      !isEditingExemplar &&
+      (this.getExemplarSettings()?.validationEnabled ?? false)
+    );
+  }
+
+  getExemplarValidationResults(): {satisfied: boolean; message: string} {
+    if (!this.shouldValidateWithExemplar()) {
+      throw new Error(
```

Comment:
Throwing an error might be a bit overkill since we'd crash the page...in theory could this happen due to a weird level configuration/error? Maybe we just return the error message and `satisfied: true`, and log something to cloudwatch to minimize user impact?